db/schema.rb

```diff
@@ -29,7 +29,7 @@
     t.text "metadata"
     t.string "service_name", null: false
     t.bigint "byte_size", null: false
-    t.string "checksum", null: false
+    t.string "checksum"
```

Comment:
Is this change supposed be included?

Reply:
Hi @danswick - db/schema.rb got updated as a part of running the DB migrations when I took a pull from the main branch.